app/components/main/main.html

```diff
@@ -1,4 +1,6 @@
 <!-- Main HTML content -->
+<script src="node_modules/shpjs/dist/shp.min.js"></script>
```

Comment:
better to inject this line in the uncompiled index.html in the app/ folder of the project.